app/models/katello/system.rb

```diff
@@ -95,11 +95,20 @@ def self.uuids_to_ids(uuids)
       systems.pluck(:id)
     end
 
-    def self.with_unavailable_errata(errata)
+    def self.with_non_installable_errata(errata)
       subquery = Katello::Erratum.select("#{Katello::Erratum.table_name}.id").installable_for_systems.where("#{Katello::SystemRepository.table_name}.system_id = #{System.table_name}.id").to_sql
       self.joins(:applicable_errata).where("#{Katello::Erratum.table_name}.id" => errata).where("#{Katello::Erratum.table_name}.id NOT IN (#{subquery})").uniq
     end
 
+    def self.with_applicable_errata(errata)
+      self.joins(:applicable_errata).where("#{Katello::Erratum.table_name}.id" => errata)
+    end
+
+    def self.with_installable_errata(errata)
+      subquery = Katello::Erratum.select("#{Katello::Erratum.table_name}.id").installable_for_systems.where("#{Katello::SystemRepository.table_name}.system_id = #{System.table_name}.id").to_sql
+      self.joins(:applicable_errata).where("#{Katello::Erratum.table_name}.id" => errata).where("#{Katello::Erratum.table_name}.id IN (#{subquery})").uniq
```

Comment:
ahhh actually, the reason it was done that for non_installable was because it was a NOT IN.  I can use your suggestion just fine